addons/website/views/snippets/snippets.xml

```diff
@@ -408,8 +408,12 @@
     <!-- Stretch section -->
     <div data-js="SectionStretch"
```

Comment:
@sdegueldre The JS option makes no sense anymore, it should be adapted.

After discussion with SBU, we decided to simply remove it. Most of the time at least "small" will have a visible effect and it's ok if "Full" and "Regular" appear to be the same in some cases. I'll do it myself so that we can merge this task now.